### PR TITLE
Update Foreman 3.9 to supported

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: web/content/js/versions.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ const navVersions = [
     ]
   },
   {
-    "title": "Foreman 3.9 - Katello 4.11 (RC)",
+    "title": "Foreman 3.9 - Katello 4.11",
```
Comment (maximiliankolb) [inline web/content/js/versions.js:203]: Please change line 519 to "unsupported".

File: web/content/index.adoc [hunk 2]
```diff
@@ -6,21 +6,18 @@ title: Home
...
 * link:/release/3.8/[Foreman 3.8 & Katello 4.10]
-* link:/release/3.7/[Foreman 3.7 & Katello 4.9]
 
 === Unsupported releases
 
 The future version is built in nightly.
 
 * link:/release/nightly/[Nightly]
 
-There is a release candidate available for testing.
```
Comment (maximiliankolb) [inline web/content/index.adoc:18]: You could also prefix this line with `// ` to keep it around as an invisible comment for the next RC.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: web/content/index.adoc (hunk 2 above)
Comment (Griffin-Sullivan) [reply web/content/index.adoc:18]: I added the comment.

**Review by maximiliankolb — APPROVED** [review]
diff LGTM!

**Review by maximiliankolb — APPROVED** [review]
diff looks even better now. :)

**Pull request merged**